crates/fda/src/main.rs

```diff
@@ -1176,20 +1177,53 @@ async fn pipeline(format: OutputFormat, action: PipelineAction, client: Client)
     }
 }
 
-async fn table_connector(
+async fn connector(
+    format: OutputFormat,
     pipeline_name: String,
-    table_name: &str,
-    connector_name: &str,
+    relation: &str,
+    connector: &str,
     action: ConnectorAction,
     client: Client,
 ) {
+    let rc = client
+        .get_pipeline()
+        .pipeline_name(pipeline_name.clone())
+        .send()
+        .await
+        .map_err(handle_errors_fatal(
+            client.baseurl.clone(),
+            "Failed to get pipeline config",
+            1,
+        ))
+        .map(|response| response.program_info.clone())
+        .unwrap()
+        .unwrap();
+
+    let full_connector_name = format!("{relation}.{connector}");
```

Comment:
I think you need to call `canonical_name()` on `relation` unless it's already canonicalized. At least that's what I had to do in integration tests

Reply:
?

Reply:
Sorry, I meant `canonical_identifier`, and I've now realized it's not needed.